lamindb_setup/core/_hub_core.py

```diff
@@ -367,6 +367,61 @@ def _connect_instance(
     return instance, storage  # type: ignore
 
 
+def _connect_instance_remote(
```

Comment:
Can you call it `_connect_instance_new()` instead?

And then we'll delete the existing `_connect_instance` function once we spend 1-2 weeks testing it.

Reply:
done